docs/native-modules-android.md

```diff
@@ -294,7 +294,6 @@ Locate ReactNativeHost’s `getPackages()` method and add your package to the pa
 override fun getPackages(): List<ReactPackage> =
     PackageList(this).packages.apply {
         // Packages that cannot be autolinked yet can be added manually here, for example:
-        // packages.add(new MyReactNativePackage());
```

Comment:
Let's update this to:
```kotlin
// add(MyReactNativePackage())
```

Reply:
Okay, what do you think about this?

In the Kotlin example:
```kotlin
override fun getPackages(): List<ReactPackage> =
    PackageList(this).packages.apply {
        // Packages that cannot be autolinked yet can be added manually here, for example:
        // add(MyAppPackage())
    }
```
In the Java example:

```java
@Override
  protected List<ReactPackage> getPackages() {
    @SuppressWarnings("UnnecessaryLocalVariable")
    List<ReactPackage> packages = new PackageList(this).getPackages();
    // Packages that cannot be autolinked yet can be added manually here, for example:
    // packages.add(new MyAppPackage());
    return packages;
  }
```